apps/system/test/unit/browser_context_menu_test.js

```diff
@@ -173,7 +176,9 @@ suite('system/BrowserContextMenu', function() {
 
     var stubStopPropagation =
       this.sinon.stub(fakeContextMenuEvent, 'stopPropagation');
-
```

Comment:
I think the mock for mozSettings has a "sync" mode. Can you use this instead of stubing an internal method?